pkg/drive/model/storage.ts

```diff
@@ -0,0 +1,3 @@
+export interface Storage {
+  upload(name: string, file: Uint8Array): Promise<void>;
```

Comment:
```suggestion
  upload: (name: string, file: Uint8Array) => Promise<void>;
```
コーディング規約で決めたように, 変性に関する型検査を強制するためにこのように書いておきたいです.